resources/views/habbo-web-pages/common/box_habbo_way.blade.php

```diff
@@ -1,2 +1,2 @@
 <h3>{!! trans('common/box_habbo_way.title', ['hotelName' => $chocolatey->hotelName]) !!}</h3>
-<p>{!! trans('common/box_habbo_way.description', ['hotelName' => $chocolatey->hotelName]) !!}</p>
\ No newline at end of file
+<p>{!! trans('common/box_habbo_way.description', ['hotelName' => $chocolatey->hotelName], ['shortName' => $chocolatey->shortName]) !!}</p>
```

Comment:
`['hotelName' => $chocolatey->hotelName], ['shortName' => $chocolatey->shortName]` this is wrong

should be 
`['hotelName' => $chocolatey->hotelName, 'shortName' => $chocolatey->shortName]`